platform/lib/platform_web/components.ex

```diff
@@ -881,9 +880,9 @@ defmodule PlatformWeb.Components do
                           project: @update.media.project
                         ) %> updated
                       <%= if not is_nil(attr) do %>
-                        <%= live_patch class: "text-button text-gray-800 inline-block", to: "/incidents/#{@update.media.slug}/history/#{attr.name}" do %>
+                        <.link class = "text-button text-gray-800 inline-block" patch = "/incidents/#{@update.media.slug}/history/#{attr.name}">
```

Comment:
To use the #{...} syntax in a string (which is an Elixir concept), you must wrap the value in {} (i.e., `patch={"..."}`).